### ISSUE TITLE
Request::sendPhoto raises "Telegram returned an invalid response"

### ISSUE THREAD

**Issue by da-rt** [php-telegram-bot/core#549]
Hi, 

- Operating system: Linux 2.6.32-openvz-042stab123.1-amd64 #1 SMP Wed Mar 22 15:22:33 
- PHP version: 5.6.30-0+deb8u1 (64bit); apache2handler; Linux
- PHP Telegram Bot version: 0.45.0 and 0.44.0
- Using MySQL database: yes 
- MySQL version: 5.5.55-0+deb8u1-log
- Update Method: Webhook or getUpdates
- Self-signed certificate: yes or no 

### Expected behaviour
Request::sendPhoto should send image, as it worked till today

### Actual behaviour
Request::sendPhoto raises "Telegram returned an invalid response! Please review your bot name and API key"
But if I use sendMessage against, all is working well.

### Extra details
I tried to create another bot, just using the same code and different apikey and all is going well, even sendPhoto method. 

So, in summary, sendPhoto doesn't work ONLY with specific bot. 

Could somebody help me? 

Thanks

**Comment by da-rt** [comment]
Strange thing, after several hours it is working normally again. I don't really think that it is some kind of limits, due to a very small number of hits. So, what was it?

**Comment by noplanman** [comment]
This could be a temporary blip from Telegram, possibly due to their update to API version 3.1.
Not sure.

I'll close off here, as it seems to be solved again, but if it keeps on happening, just reopen here 👍

**Comment by noplanman** [comment]
@arc92 What version of the bot are you using?
